# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lualdap/lualdap-5.1-compat.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- lualdap/src/lualdap.c	2006/04/04 20:11:51	1.44
+++ lualdap/src/lualdap.c	2006/07/24 01:36:51	1.45
@@ -1,7 +1,7 @@
 /*
 ** LuaLDAP
 ** See Copyright Notice in license.html
-** $Id: lualdap.c,v 1.44 2006/04/04 20:11:51 carregal Exp $
+** $Id: lualdap.c,v 1.45 2006/07/24 01:36:51 tomas Exp $
 */
 
 #include <stdlib.h>
@@ -16,8 +16,9 @@
 
 #include "lua.h"
 #include "lauxlib.h"
+#if ! defined (LUA_VERSION_NUM) || LUA_VERSION_NUM < 501
 #include "compat-5.1.h"
-
+#endif
 
 #define LUALDAP_PREFIX "LuaLDAP: "
 #define LUALDAP_TABLENAME "lualdap"

